rules/awsrules/aws_resource_missing_tags.go

```diff
@@ -279,30 +287,41 @@ func (r *AwsResourceMissingTagsRule) Link() string {
 
 // Check checks resources for missing tags
 func (r *AwsResourceMissingTagsRule) Check(runner *tflint.Runner) error {
-	attributeName := "tags"
 	config := awsResourceTagsRuleConfig{}
 	if err := runner.DecodeRuleConfig(r.Name(), &config); err != nil {
 		return err
 	}
 
 	for _, resourceType := range r.resourceTypes {
+		// Skip this resource if its type is excluded in configuration
+		if stringInSlice(resourceType, config.Exclude) {
+			continue
+		}
+
+		// Special handling for tags on aws_autoscaling_group resources
+		if resourceType == "aws_autoscaling_group" {
+			r.checkAwsAutoScalingGroups(runner, config)
```

Comment:
The returned error appears to be discarded. Please use the `EnsureNoError` API to test if you should stop the inspection because this error can be application-specific errors.

Reply:
Oops, I had meant to come back to that but forgot. I will emit a message with `err.Error()`.

Reply:
d583da6